x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_definition/apm_settings.ts

```diff
@@ -280,27 +220,9 @@ function getApmSettings(isCloudPolicy: boolean): SettingDefinition[] {
   ];
 }
 
-export function APMSettingsForm({ vars, onChange, isCloudPolicy }: Props) {
-  const apmSettings = useMemo(() => {
-    return getApmSettings(isCloudPolicy);
-  }, [isCloudPolicy]);
-
-  return (
-    <SettingsForm
-      title={i18n.translate(
-        'xpack.apm.fleet_integration.settings.apm.settings.title',
-        { defaultMessage: 'General' }
-      )}
-      subtitle={i18n.translate(
-        'xpack.apm.fleet_integration.settings.apm.settings.subtitle',
-        { defaultMessage: 'Settings for the APM integration.' }
-      )}
-      settings={apmSettings}
-      vars={vars}
-      onChange={(key, value) => {
-        const newVars = mergeNewVars(vars, key, value);
-        onChange(newVars, isSettingsFormValid(apmSettings, newVars));
-      }}
-    />
-  );
+export function isAPMFormValid(
+  newVars: PackagePolicyVars,
+  apmSettings: SettingDefinition[]
+) {
+  return isSettingsFormValid(apmSettings, newVars);
 }
```

Comment:
What's the purpose of this wrapper? 

```suggestion
export function isAPMFormValid(
  newVars: PackagePolicyVars,
  apmSettings: SettingDefinition[]
) {
  return isSettingsFormValid(apmSettings, newVars);
}
```

Reply:
Yeah, no need, I'll call it direct at  public/components/fleet_integration/apm_policy_form/index.tsx